SpatialGDK/Source/SpatialGDKTests/Private/RPCContainer/ObjectDummy.cpp

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "ObjectDummy.h"
+
+#include <Core.h>
```

Comment:
Unreal codebase doesn't really use `#include <Foo.h>`. Please stick with `#include "Foo.h"` across this PR.

Reply:
Yes, I see UnrealGDK plugin contains code that mostly uses #include "", so I can put quotes back to preserver consistence.

Reply:
Why don't we use the more usual <> for code we didn't write. If it's just to copy unreal and not for a reason I'd suggest we break with that.